packages/jwt/README.md

```diff
@@ -0,0 +1,32 @@
+# `@byu-oit/jwt`
+
+> Provides helpful functions to retrieve a specified BYU .well-known URL and verify BYU signed JWTs.
+
+## Usage
+
+```javascript
+import {ByuJwt} from '@byu-oit/jwt'
+
+const byuJwt = new ByuJwt({ issuer: 'https://api.byu.edu' })
+
+const jwt = byuJwt.verify('[your jwt]')
+
+/** Access the jwt payload information */
+const { byuId } = jwt.payload
+
+/** Access the jwt header information */
+const { alg } = jwt.header
+```
+
+> **Note**
+> Please refer to the API documentation if you need to see what information is made available in the jwt payload or header.
```

Comment:
Link, please. 🙂

Reply:
Kind of a Chicken VS Egg problem, but I had the same thought. We'll create an issue for it so we don't forget to add it once we have the documentation deployed.

Reply:
Captured as #90. Marking this as resolved.